src/Symfony/Component/Routing/Tests/Loader/AnnotationClassLoaderTest.php

```diff
@@ -11,35 +11,45 @@
 
 namespace Symfony\Component\Routing\Tests\Loader;
 
-use Symfony\Component\Routing\Annotation\Route;
-
-class AnnotationClassLoaderTest extends AbstractAnnotationLoaderTest
+use Doctrine\Common\Annotations\AnnotationReader;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use LogicException;
```

Comment:
oups, we missed this file, it's behind on CS, please revert them and keep only the relevant changes for this PR

Reply:
Seriously, we should really really really have a CS fixer which does this. It's SOOOOOO BORING and doesn't add any value, just frustration. All this code hasn't changes in 4 years, there's no TLS that has had any change in them, the likeliness of this changing again in older supported version is just so small.